src/schemaGenerator.ts

```diff
@@ -637,6 +647,49 @@ function runAtMostOncePerRequest(
   };
 }
 
+/**
+ * Attach directive resolvers to Schema
+ * https://github.com/apollographql/graphql-tools/issues/212#issuecomment-324447078
+ *
+ * @author Agon Bina <agon_bina@hotmail.com>
```

Comment:
We usually don't track authors in this way, since we don't have them for any other functions. Would you prefer to have some attribution other than just Git/GitHub's tracking of committers?

Reply:
Removed